ktlint/src/main/kotlin/com/pinterest/ktlint/Main.kt

```diff
@@ -425,7 +436,11 @@ object Main {
                         }
                     }
                     ?: emptyMap<String, String>()
-                ) + cliUserData + ("file_path" to fileName)
+                ) +
+                cliUserData +
+                // For now, use a global list
+                ("disabled_rules" to disabledRules.joinToString(",")) +
```

Comment:
Here is where we'd handle logic around a .ktlintignore file that's formatted more like an .editorconfig file.